### PR TITLE
Miscellaneous fixes for the firefox-ci export job

### PR DESCRIPTION
**Pull request by ahal**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is
  referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to
  `.circleci/config.yml`) will cause environment variables (particularly
  credentials) to be exposed in test logs
- [ ] Ensure the container image will be using permissions granted to
  [telemetry-airflow](https://github.com/mozilla/telemetry-airflow/)
  responsibly.

### CODE REVIEW

File: jobs/fxci-taskcluster-export/fxci_etl/pulse/handler.py [hunk 1]
```diff
@@ -91,8 +91,14 @@ def __init__(self, config: Config, **kwargs: Any):
 
     def process_event(self, event):
         data = event.data
+
+        if data.get("runId") is None:
+            # This can happen if `deadline` was exceeded before a run could
```
Comment (bhearsum) [inline jobs/fxci-taskcluster-export/fxci_etl/pulse/handler.py:96]: To be clear: we can still have deadline exceeded with a `runId` (in which case we still process it, which is what we want)?

**Review by bhearsum — CHANGES_REQUESTED** [review]
It looks like you need to drop the `table_name` methods in https://github.com/mozilla/docker-etl/blob/main/jobs/fxci-taskcluster-export/fxci_etl/pulse/records.py as well?

File: jobs/fxci-taskcluster-export/fxci_etl/pulse/handler.py (hunk 1 above)
Comment (ahal) [reply jobs/fxci-taskcluster-export/fxci_etl/pulse/handler.py:96]: I'm not sure. I didn't come across this case in the wild, but I noticed that the TC docs mentioned that `runId` might not always be set and gave `deadline-exceeded` as an example of when that would happen.

For now, I think it's fine to ignore events where it's unset. Tasks that never ran don't have a cost.

**Comment by ahal** [comment]
Thanks for catching those missed `table_name` methods!

File: jobs/fxci-taskcluster-export/fxci_etl/pulse/handler.py (hunk 1 above)
Comment (bhearsum) [reply jobs/fxci-taskcluster-export/fxci_etl/pulse/handler.py:96]: I certainly agree that it's safe to ignore anything without a `runId`, for the record! This was mostly a note that `deadline-exceeded` may also happen when a `runId` is present, AIUI.

**Review by bhearsum — APPROVED** [review]

**Pull request merged**